CondTools/BeamSpot/test/BeamProfile2DBWriter_cfg.py

```diff
@@ -0,0 +1,59 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("writeBeamProfile2DB")
+
+options = VarParsing.VarParsing()
+options.register('unitTest',
+                 False, # default value
+                 VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                 VarParsing.VarParsing.varType.bool, # string, int, or float
+                 "are we running the unit test?")
+options.register('inputTag',
+                 "myTagName", # default value
+                 VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                 VarParsing.VarParsing.varType.string, # string, int, or float
+                 "output tag name")
+options.parseArguments()
+
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+from CondCore.CondDB.CondDB_cfi import *
+
+if options.unitTest :
+    tag_name = 'simBS_tag'
+else:
+    tag_name = options.inputTag
+
+#################################
+# Produce a SQLITE FILE
+#################################
+CondDBSimBeamSpotObjects = CondDB.clone(connect = cms.string('sqlite_file:test_%s.db' % tag_name)) # choose an output name
+process.PoolDBOutputService = cms.Service("PoolDBOutputService",
+                                          CondDBSimBeamSpotObjects,
+                                          timetype = cms.untracked.string('runnumber'),
+                                          toPut = cms.VPSet(cms.PSet(record = cms.string("SimBeamSpotObjectsRcd"), # SimBeamSpot record
+                                                                     tag = cms.string(tag_name))),                 # choose your favourite tag
+                                          loadBlobStreamer = cms.untracked.bool(False)
+                                          )
+
+process.source = cms.Source("EmptySource")
+
+process.maxEvents = cms.untracked.PSet(input = cms.untracked.int32(1))
+
+process.simbeamspotwriter = cms.EDAnalyzer("BeamProfile2DB",
```

Comment:
Now that you have a properly auto generated `cfi` file you can use it here.

Reply:
addressed in [a51ddf1](https://github.com/cms-sw/cmssw/pull/41918/commits/a51ddf14d10f6ad29ca043ce717968aa496f9c90)